source/develop/release-management/features/ovirt-ovn-provider.md

```diff
@@ -0,0 +1,334 @@
+the nic must be added to the OVN integration bridge.
+This is task is performed automatically by libvirt, if the VNIC xml
+is appropriately modified.
+This is done by adding the following section to the xml:
+
+         <source bridge='ovsbr0'/>
+         <virtualport type='openvswitch'>
+           <parameters interfaceid='<OVN logical switch port name>'/>
+         </virtualport>
+
+The value of the "bridge" attribute of the "source" element is the OVS switch to which
+the VNIC should be added.
+The  "interfaceid" attribute of the "virtualport/parameters" element is the name
+of the OVN logical switch port which should be associated with this VNIC, and sets
+the value of the "external-ids:iface-id" column in the logical switch port record.
+The complete xml would look as follows:
+
+    <interface type='bridge'>
+         ...
+         <source bridge='ovsbr0'/>
+         <virtualport type='openvswitch'>
+           <parameters interfaceid='<OVN logical switch port name>'/>
+         </virtualport>
+         ...
+    </interface>
+
+This is the equivalent of executing the following OVS commands:
+	ovs-vsctl add-port br-int <nic name> -- set Interface <nic name> external_ids:iface-id=<OVN logical switch port name>
+
+The "external-ids:iface-id" parameter allows OVN controler to associate 
+this nic with the logical port defined in the southbound db. 
+OVN controller will update the OVS OpenFlow tables in accordance with
+the southbound db.
+The binding in the southbound db is updated with the chassis id.
+
+OVN controllers on different hosts notice the updated chassis id in the binding,
+and knowing the physical location of the port update local ovs flows.
+
+#### Detailed steps of nic plugging
+
+* the engine checks if the port for the VM NIC exists, issuing a GET port request to the provider. The provider
+in turn queries the Logical_Switch_Port table of the OVN north DB
+*   if no port exists, the engine will create a new port by issuing a POST port request to the provider. The provider
+    in turn adds a row to the Logical_Switch_Port table of the OVN north DB	
+*   if a port already exists, the engine will update the port by issuing a POST port request to the provider. The provider
+    in turn updates the port row in the Logical_Switch_Port table of the OVN north DB
+*   in both cases the new/updated port is identified by a <PORT ID>, which is the uuid of the port row in the Logical_Switch_Port table of the OVN north DB
+*   the engine will send a nic plug request to VDSM, passing the <PORT ID>
+    as one of the parameters. The <PORT ID> is passed to the VIF driver (VDSM hook) as  a "vnic_id" parameter.
+*   on VDSM, the VIF Driver, invoked using the VDSM before_nic_hotplug/before_device_create hook,
+    will connect the VM NIC to the network provided by the external provider by modifing the device xml
+
+![](external_network_provider_schema1.png "fig:external_network_provider_schema1.jpg")
```

Comment:
png/jpg seems inconsistent. I think think you should drop the jpg altogether. any chance to using svg?